### PR TITLE
feat: enable hebrew locale

### LINKED ISSUE #4867: Add support for Hebrew locale
**What you want IINA to do:**
Support the Hebrew locale.

**Why you think this should be added:**
Hebrew translations are now 99% complete in [IINA's instance of Crowdin](https://translate.iina.io/).

### PR DESCRIPTION
**Pull request by ShlomoCode**
- [x] I have read [CONTRIBUTING.md](https://github.com/iina/iina/blob/develop/CONTRIBUTING.md)
- [x] This implements/fixes issue #4867.

---

**Description:**

The Hebrew translation is complete (100% of the strings), this PR activates the language.
This PR is a replacement of PR #4864 that was closed by mistake.

### CODE REVIEW

File: iina.xcodeproj/project.pbxproj
Comment (low-batt) [inline iina.xcodeproj/project.pbxproj:526]: The `KeyBinding.strings` file has been translated, but is missing from Xcode. As a result the names shown for keys in the `Key Bindings` settings tab are in English.

File: iina.xcodeproj/project.pbxproj
Comment (low-batt) [inline iina.xcodeproj/project.pbxproj:527]: The `InfoPlist.strings` file is also missing from Xcode. The file is present, but is empty. I show the Italian file for comparison:
```text
low-batt@gag iina (he *$)$ cat he.lproj/InfoPlist.strings 
low-batt@gag iina (he *$)$ 
low-batt@gag iina (he *$)$ cat it.lproj/InfoPlist.strings 
/* (No Comment) */
"NSHumanReadableCopyright" = "Distribuito sotto licenza GPLv3";
low-batt@gag iina (he *$)$ 
```

**Review by low-batt — CHANGES_REQUESTED** [review]
Two minor issues. See inline comments.

**Comment by low-batt** [comment]
I took another look at this PR as we are currently discussing merging it. I noticed the following discrepancy. Xcode reports 35 files localized for English, but only 33 files localized for Hebrew. Many other languages only report 34 files localized so the issue is not just with the Hebrew translations:
count

I traced the difference to the two files I mention in the inline comments.

**Comment by ShlomoCode** [comment]
@low-batt The translations here on GitHub are very far from the version on crowdin. Someone with permission in crowdin (@uiryuu?) should update the files here, then I can handle the review comments here

BTW, I don't think it's worth merging this PR before merging all the other PRs that fix the design problems in RTL

**Comment by uiryuu** [comment]
@ShlomoCode I'll check the discrepancies between GitHub and Crowdin. 

We are considering merge this PR first to better inspect all the RTL-related issues. If we find the next release version is not ready for RTL, we will disable HE language manually in the next release.

File: iina.xcodeproj/project.pbxproj
Comment (uiryuu) [reply iina.xcodeproj/project.pbxproj:527]: I checked `Base.lproj` and `en.lproj` for this file, both of them are empty. That's why Crowdin doesn't have this string either. I'll add this string to base and en so that crowdin can pick it up.

**Comment by uiryuu** [comment]
@ShlomoCode I checked Corwdin, and it seems to be synced with the latest develop branch. Can you point out the discrepancies you noticed between Corwdin and dev branch?

File: iina.xcodeproj/project.pbxproj
Comment (uiryuu) [reply iina.xcodeproj/project.pbxproj:526]: This should be a simple fix; I can manually enable this file after merging this PR.

Comment (ShlomoCode) [reply iina.xcodeproj/project.pbxproj:526]: Done

File: iina.xcodeproj/project.pbxproj
Comment (ShlomoCode) [reply iina.xcodeproj/project.pbxproj:527]: https://github.com/iina/iina/pull/4937

Comment (ShlomoCode) [reply iina.xcodeproj/project.pbxproj:527]: @uiryuu I see you fixed it at https://github.com/iina/iina/pull/4937. Does it block this PR?

**Comment by ShlomoCode** [comment]
> Can you point out the discrepancies you noticed between Corwdin and dev branch?

Quick example: https://crowdin.com/editor/iina/19/en-he?view=comfortable#237
Changed 16 days ago, not applied to Github

**Comment by low-batt** [comment]
A side issue. I confirmed the `Key Bindings` settings are translated now, but there are some strings that still need to be translated:
mostly-fixed


It is the two entries at the end of the `KeyBindings.string` file:
```text
"read.cycle-values" = "עבור בין {{property}} ב({{{values}}})";
"read.seek" = "Seek {{#fwd}}forward{{/fwd}}{{#bwd}}backward{{/bwd}}{{#abs}}to{{/abs}} {{value}}{{#sec}}s{{/sec}}{{#per}}%{{/per}} {{#exact}}(exact){{/exact}}{{#kf}}(keyframe){{/kf}}";
"read.sub-seek" = "Seek to {{#skip_1}}next{{/skip_1}}{{#skip_-1}}previous{{/skip_-1}} subtitle";
```

**Comment by ShlomoCode** [comment]
> "read.seek" = "Seek {{#fwd}}forward{{/fwd}}{{#bwd}}backward{{/bwd}}{{#abs}}to{{/abs}} {{value}}{{#sec}}s{{/sec}}{{#per}}%{{/per}} {{#exact}}(exact){{/exact}}{{#kf}}(keyframe){{/kf}}";
> "read.sub-seek" = "Seek to {{#skip_1}}next{{/skip_1}}{{#skip_-1}}previous{{/skip_-1}} subtitle";

It was translated a long time ago...
https://crowdin.com/editor/iina/27/en-he?view=comfortable#687
https://crowdin.com/editor/iina/27/en-he?view=comfortable#689
I have no idea why this doesn't affect the file on github. @uiryuu?

**Comment by uiryuu** [comment]
@ShlomoCode I'll sync the newly translated string to repo. You'll see the updated translations soon.

**Comment by uiryuu** [comment]
This PR has been merged to iina:enable-he branch. Later, with all the RTL-related PR also merged into enable-he branch, the branch will be merged into dev. @ShlomoCode Thanks for your effort for making this possible!

**Comment by ShlomoCode** [comment]
I'm curious to understand why you prefer to merge all to iina:enable-he branch as an intermediate step and not directly to dev

**Comment by uiryuu** [comment]
We are not sure whether or not to include RTL into 1.3.5 release or 1.4.0 beta releases, so we hold off merging them into dev branch now. If all the RTL-related issues are resolved before 1.3.5 release, we will include HE locale in 1.3.5; otherwise it goes into 1.4.0 beta releaes.

**Comment by ShlomoCode** [comment]
> @ShlomoCode I'll sync the newly translated string to repo. You'll see the updated translations soon.

In which branch do I find it?

**Comment by low-batt** [comment]
Branch `enable-he`. It is listed at the top of this page after the `Merged` button.

**Comment by ShlomoCode** [comment]
I don't see the translations sync commit
CleanShot 2024-05-24 at 15 51 11@2x

**Comment by low-batt** [comment]
I had noticed that as well. Need to ask @uiryuu about that. I don't know how to investigate that.

I'm not seeing nightly builds, so maybe an issue with our server?

**Comment by uiryuu** [comment]
@ShlomoCode The Crowdin integration is not working now. Im still struggling at finding a solution.

**Comment by uiryuu** [comment]
@ShlomoCode Fixed the Crowdin integration and rebased `enable-he` to the latest dev branch. Now you can see the latest translations on the `enable-he` branch.

**Pull request merged**
